sdk/storage/azure-storage-file-datalake/azure/storage/file/out/datalake/_data_lake_file_client.pyi

```diff
@@ -0,0 +1,17 @@
+# Stubs for datalake._data_lake_file_client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from azure.storage.file.datalake._path_client import PathClient
+from typing import Any, Optional
+
+class DataLakeFileClient(PathClient):
+    def __init__(self, account_url: str, file_system_name: str, file_directory: str, file_name: str, credential: Optional[Any]=..., **kwargs: Any) -> Any: ...
```

Comment:
should we make file_directory optional? If the user wants to create a file under root directory, are they supposed to pass None for file_directory?

Reply:
We removed file_directory parameter currently and ask them to pass a full file path instead